File: 2015/java/src/test/java/nl/sbmf21/aoc15/days/Day14Test.java
package nl.sbmf21.aoc15.days;

import org.junit.jupiter.api.Test;

import static nl.sbmf21.aoc.testing.UtilKt.testDay;

public class Day14Test {

    @Test
    public void testInput() {
        Day14.REINDEER_SCORE_TIME = Day14.DEFAULT_REINDEER_SCORE_TIME;
        testDay(Day14.class, 2640, 1102, false, null);
    }

    @Test
    public void testExample() {
        Day14.REINDEER_SCORE_TIME = 1000;
        testDay(Day14.class, 1120, 689, true, null);
    }
}

File: 2022/kotlin/src/test/kotlin/nl/sbmf21/aoc22/days/Day14Test.kt
package nl.sbmf21.aoc22.days

import nl.sbmf21.aoc.testing.testDay
import org.junit.jupiter.api.Test

class Day14Test {

    @Test
    fun testInput() = testDay(Day14::class.java, 1406, 20870)

    @Test
    fun testExample() = testDay(Day14::class.java, 24, 93, true)
}